Argo/Types/StandardTypes.swift

```diff
@@ -58,12 +118,37 @@ extension Float: Decodable {
 }
 
 public extension Optional where Wrapped: Decodable, Wrapped == Wrapped.DecodedType {
+  /**
+    Decode `JSON` into an `Optional<Wrapped>` value where `Wrapped` is `Decodable`.
+
+    Returns a decoded optional value from the result of performing `decode` on
+    the internal wrapped type.
+
+    - parameter j: The JSON value to decode
+
+    - returns: A decoded optional `Wrapped` value
+  */
   static func decode(j: JSON) -> Decoded<Wrapped?> {
     return .optional(Wrapped.decode(j))
   }
 }
 
 public extension CollectionType where Generator.Element: Decodable, Generator.Element == Generator.Element.DecodedType {
+  /**
+    Decode `JSON` into an array of values where the elements of the array are
+    `Decodable`.
+
+    If the `JSON` is an array of `JSON` objects, this returns a decoded array
+    of values by mapping the element's `decode` function over the `JSON` and
+    then `sequencing` the result. This makes this `Decode` function an
```

Comment:
is this is in backticks because you want to refer to the function then should it be `sequence`ing?